packages/veritone-react-common/src/components/FormBuilder/FormBuilder.js

```diff
@@ -131,7 +132,7 @@ function FormBuilderComponent({
               <NullState
                 titleText="Begin by selecting block to your rignt"
               >
-                <Link href="https://help.veritone.com/en/" target= "_blank">
+                <Link href="https://help.veritone.com/en/" target="_blank">
```

Comment:
Are we going to use this url?

Reply:
It's just a placeholder href. We actually can make it inside a config

Reply:
I mean is this url going to point general help site or specific help page @mrgorji ?

Reply:
We actually do not have specific help page @MmtBkn